### ISSUE TITLE
Glyph column width should use the largest glyph width, not xAdvance

### ISSUE THREAD

**Issue by petrbrouzda** [tchapi/Adafruit-GFX-Font-Customiser#27]
When I extract font with big characters (attached), the UI is rendered incorrectly and font can't be edited.
Chrome, Windows 10

![image](https://user-images.githubusercontent.com/9107312/120377916-0b3e4e80-c31e-11eb-9fa9-6274352874d1.png)

Font file:
[IconsFont2.h.txt](https://github.com/tchapi/Adafruit-GFX-Font-Customiser/files/6580102/IconsFont2.h.txt)

**Comment by tchapi** [comment]
Oh ! It seems like the width calculation is off for the glyph columns

**Comment by tchapi** [comment]
I'll look into that, good catch

**Comment by tchapi** [comment]
The calculation is based on the max `xAdvance` figure found in the font. Is there any reason that this font only has a max of `4` pixels for that property ?

You can see the `xAdvance` here on this glyph: it's the second vertical red line. In a conventional font, it indicates how subsequent characters overlap:

![Screenshot 2021-06-01 at 21 22 41](https://user-images.githubusercontent.com/1944007/120379164-a683f380-c31f-11eb-8e1c-40b2f9d88ee0.png)

So basically if you use this font as-is and use two characters on a line, they will totally overlap (except for 4 pixels) which doesn't seem right.

While I think I should take into account the actual max width of a glyph instead in this tool, I think the `xAdvance` for the characters of this font is not correctly set up.

What do you think ?

**Comment by petrbrouzda** [comment]
I didn't care about the xAdvance, because I'm using this font as iconset only - so I print only one character at a time, never more than one.
So I'm able to easily solve the problem by manualy editing xAdvance of one character - no fix from you is needed.
Thank you.

**Comment by tchapi** [comment]
Still I think that the width should be calculated better. I have made a fix in a37d9d4, tell me if that still works well for you !

**Comment by petrbrouzda** [comment]
Works correctly now - even on a font with bad xAdvance. Thank you.
